sdk/data/azcosmos/cosmos_global_endpoint_policy.go

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package azcosmos
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore/policy"
+)
+
+// globalEndpointManagerPolicies contains policies related to caching and the global endpoint manager.
+type globalEndpointManagerPolicies struct {
+	endpointMgr *globalEndpointManager // endpointMgr is a pointer to the global endpoint manager instance.
+}
+
+// NewGlobalEndpointManagerPolicies creates a new instance of globalEndpointManagerPolicies.
+func NewGlobalEndpointManagerPolicies(endpointMgr *globalEndpointManager) *globalEndpointManagerPolicies {
```

Comment:
This should be lowe camel case, otherwise it's public.

```suggestion
func newGlobalEndpointManagerPolicy(endpointMgr *globalEndpointManager) *globalEndpointManagerPolicies {
```